packages/website/src/components/Navbar/Navbar.tsx

```diff
@@ -24,11 +25,16 @@ export const Navbar = () => {
         position="relative"
       >
         {baseNavbar}
-        <Flex>
-          <Button size={buttonSize} mr="12px" onClick={() => openModal('mentor')}>
+        <Flex overflow="hidden">
+          <Button fontSize={isSmallerThan370px ? 'xs' : 'md'} size="md" mr="12px" onClick={() => openModal('mentor')}>
```

Comment:
🤔 We shouldn't modify the font size of our buttons as it creates inconsistency in our design system